website/data/dd.md

```diff
@@ -68,6 +68,9 @@ This example was generated from synthetic claims data. Please expand the source
  {
    "resourceType": "ExplanationOfBenefit",
    "id": "snf-777777777",
+   "meta": {
+        "lastUpdated": "2019-05-25T23:45:16.962-04:00"
```

Comment:
Is 5-25 the anticipated release date?

Reply:
The earliest date is 2020-02-13T00:00:00.000-05:00 that you can search for. This may have come from the synthetic data or I may have made it up. Not sure. Do you have a preference? Maybe after the earliest date?

Reply:
I think the 2-13 date makes sense